tools/importer-rest-api-specs/featureflags/constants.go

```diff
@@ -6,4 +6,4 @@ package featureflags
 const AllowConstantsWithoutXMSEnum = true
 
 // GenerateJsonDataAPI specifies whether the Data API in JSON should be generated.
-const GenerateJsonDataAPI = false
```

Comment:
Can we keep this turned off for now - at least until we've done a manual review of it together e.g. generate a couple of services/scenarios and take a look at the output.

Reply:
💯